src/main/java/internetshop/Generator.java

```diff
@@ -0,0 +1,36 @@
+package internetshop;
+
+public class Generator {
+    private static Long itemId;
+    private static Long userId;
+    private static Long bucketId;
+    private static Long orderId;
+
+    public static Long genItemId() {
```

Comment:
```suggestion
    public static Long getItemId() {
```
or
так буде більш краще

Reply:
ok, thanks,  i will change it in next commit